bin/cyhy-import

```diff
@@ -79,6 +81,27 @@ def has_intersections(db, nets, filename, owner):
         print_intersections(intersections)
         return True
 
+def has_restricted_ips(nets):
+    geo_loc_db = GeoLocDB()
+    restricted_dict = defaultdict(lambda: netaddr.IPSet())
+    for ip in nets:
+        is_restricted, country = geo_loc_db.check_restricted_ip(ip)
+        if is_restricted:
+            restricted_dict[country].add(ip)
+            # We can't just exit here since check_restricted_cidr()
+            # prints output for each restricted CIDR block and we
+            # want the user to see all restricted CIDR blocks
+            # without having to run multiple times.
+    if len(restricted_dict) > 0:
```

Comment:
I personally prefer the more specific code that is already present, although @mcdonnnj's suggestion is more Pythonic.  I'm OK with either way.  Is this preference addressed (possibly via a link to another document) in our development guide docs?

Reply:
The development guide doesn't say much about actual Python code preferences. When dealing with things like this I prefer to defer to the language community's customs so that you can leverage optimizations and have code that appears similar to other code written in the same language.